### PR TITLE
Add argilla chatbot blogpost

### PR DESCRIPTION
**Pull request by plaguss**
## Description

This PR adds a blogpost for a chatbot to the new Argilla 2.0 SDK (repository for reference: [argilla-sdk-chatbot](https://github.com/argilla-io/argilla-sdk-chatbot/tree/main)).

CC: @osanseviero

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -4319,3 +4319,16 @@
     - spaces
     - inference
     - hardware
+
+- local: argilla-chatbot
+  title: "Creating a chatbot for the new Argilla SDK: leveraging distilabel to fine-tune a custom Embedding model for RAG"
+  thumbnail: /blog/assets/argilla-chatbot/thumbnail.png
+  author: plaguss
+  date: Jul 10, 2024
```
Comment (osanseviero) [inline _blog.yml:4327]: To be updated

File: argilla-chatbot.md [hunk 2]
```diff
@@ -0,0 +1,1324 @@
...
+thumbnail: /blog/assets/argilla-chatbot/thumbnail.png
+authors:
+- user: plaguss
+- user: gabrielmbmb
+- user: sdiazlor
+- user: osanseviero
+- user: dvilasuero
+---
+
+# Creating a chatbot for Argilla 2.0: Leveraging distilabel to fine-tune a custom Embedding model for RAG
```
Comment (osanseviero) [inline argilla-chatbot.md:12]: Shorter titles tend to work better

**Review by osanseviero — APPROVED** [review]

**Comment by plaguss** [comment]
The conflicts are solved and the title was shortened 😄

File: argilla-chatbot.md [hunk 3]
```diff
@@ -0,0 +1,1324 @@
+---
+title: "Argilla 2.0 Chatbot: Leveraging Distilabel to fine-tune a custom embedding model"
```
Comment (davidberenstein1957) [inline argilla-chatbot.md:2]: perhaps a domain-specific embedding model might be more catchy. WDYT?

**Pull request merged**